modules/ROOT/pages/_partials/task-view-governed-apis-for-rulesets.adoc

```diff
@@ -0,0 +1,29 @@
+// Used in docs-exchange/asset-details.adoc and docs-api-governance-documentation/view-conformance-status-in-exchange.adoc
+
+You can view the list of centrally-governed APIs to which a ruleset applies. Depending on the access you have to the API assets in Exchange, you may see only the APIs that are conformant to the ruleset or you may see APIs that are conformant and not conformant to the ruleset. For example, asset or organization administrators or contributors can see APIs in both conformance statuses. 
+
+Developers can use the *Governed APIs* list to help them improve the quality of the rulesets and APIs they develop as follows: 
+
+* When their APIs fail a ruleset, they can see which other APIs are successfully conforming to the ruleset so that they can fix their APIs.
+* When they are discovering available rulesets to use for their API, they can see which APIs are using a specific ruleset to evaluate how helpful it is.
+
+API consumers can use the *Governed APIs* list when they need to create an API service or application that is conformant to a certain set of standards or rules. They can:
+
+* Ensure there is a ruleset in Exchange that has the rules they need
+* See which APIs are conformant to that ruleset so that they can make sure their application is consuming only the conformant APIs
+
+To view the governed API list for a ruleset:
+
+. Select a ruleset asset that is used in an API Governance profile. 
+. Select *Governed APIs* in the navigation bar to see the following information for each API listed:
+
+* Versions of APIs governed by the ruleset in one or more API Governance profiles
+* Whether the API is governed by multiple centralized rulesets and by how many  
+* APIs that passed all of the rules in the ruleset and if you have sufficient access, those that failed
```

Comment:
I find this list kind of confusing ...mentioning apis again in the list where I think we are refering to individual apis, is "confusing"
as far as I know it will be a list of api versions, filtered by access, 
and for each  you will see, 
* Whether the API is governed by multiple centralized rulesets and by how many 
* whether the API passed or not the ruleset 
* Whether the API if conformant or not to all rulesets for which is governed.

Reply:
Good point. I've taken your suggestions.